common/libs/signing.ts

```diff
@@ -1,8 +1,15 @@
 import EthTx from 'ethereumjs-tx';
-import { ecsign, sha3 } from 'ethereumjs-util';
+import { ecsign, ecrecover, publicToAddress } from 'ethereumjs-util';
+import sha3 from 'solidity-sha3';
 import { RawTransaction } from 'libs/transaction';
 import { isValidRawTx } from 'libs/validators';
 
+export interface SignatureArgs {
+  r: Buffer;
+  s: Buffer;
+  v: number;
```

Comment:
This should also be a buffer

Reply:
Should it? It appears that in ethereum-js it is returned as a number? Or am i wrong?